requirements.txt

```diff
@@ -1,5 +1,5 @@
 scipy>=1.4.1
-scikit-learn>=0.21.3
+scikit-learn>=0.22.0
```

Comment:
Any specific reason for this?

Reply:
smac wants >22.
https://github.com/automl/SMAC3/blob/d5b93811aa74e0513e580576ebb0fbf857e18f1a/requirements.txt#L6

Reply:
Okay, I see. Thx